conf/messages

```diff
@@ -10,11 +10,14 @@ personal-details.firstname.required=Enter your first name.
 personal-details.lastname=Last name
 personal-details.lastname.required=Enter your last name.
 personal-details.nino=National Insurance number
-personal-details.nino.hint=It''s on your National Insurance card, benefit letter, payslip or P60. For example, QQ123456C
+personal-details.nino.hint=It''s on your National Insurance card, benefit letter, payslip or P60.<br />For example, QQ123456C
```

Comment:
Another way to do this to break the message into two parts.
personal-details.nino.hint.1
personal-details.nino.hint.2

Personally, I don't prefer html code in messages file. I leave it upto you if you want to change it.